dev-python/pandas/pandas-0.23.3.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="A powerful data analysis library for Python"
+DESCRIPTION="Pandas is a Python package providing fast, flexible \
+and expressive data structures designed to make working with \
+\"relational\" or \"labeled\" data both easy and intuitive. \
+It aims to be the fundamental high-level building block for \
+doing practical, real world data analysis in Python."
+HOMEPAGE="https://pandas.pydata.org/"
+COPYRIGHT="2008-2012 AQR Capital Management, LLC
+	Lambda Foundry, Inc.
+	PyData Development Team"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://files.pythonhosted.org/packages/3e/56/82c4d4c049294f87ebd05b65fdcbc9ed68bd23fb0a7e4469caf9a75d199f/pandas-0.23.3.tar.gz"
+CHECKSUM_SHA256="9cd3614b4e31a0889388ff1bd19ae857ad52658b33f776065793c293a29cf612"
+SOURCE_DIR="pandas-$portVersion"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	setuptools_python
+	setuptools_python3
+	cython_python
+	cython_python3
+	python_numpy
+	python3_numpy
+	dateutil_python
+	dateutil_python3
+	pytz_python
+	pytz_python3
+	"
+BUILD_PREREQUIRES="
+	cmd:python
+	cmd:python3
+    cmd:gcc
+	cmd:gfortran
+	"
+
+PYTHON_PACKAGES=(python python3)
+PYTHON_VERSIONS=(2.7 3.6)
```

Comment:
@bach5000: **_If_** you add support for x86 secondary arch to the pandas recipe then you will need to change the way `PYTHON_PACKAGES` and `PYTHON_VERSIONS` are set.
The correct way to do that (for recipes which need x86 secondary arch support) is to use the method shown in:
https://github.com/haikuports/haikuports/blob/b8367f1831fb639c6a40da116b76b9fe71e3ab6e/dev-python/cryptography/cryptography-2.4.1.recipe#L36